similarity-data-impl/src/main/java/org/phenotips/data/similarity/script/ExomiserViewScriptService.java

```diff
@@ -89,36 +89,56 @@ public boolean hasGenotype(Patient patient)
      */
     public JSONArray getTopGenesAsJSON(Patient patient, int g, int v)
     {
-        JSONArray result = new JSONArray();
-        if (patient == null || g <= 0) {
-            return result;
+        JSONArray variantsJSON = new JSONArray();
+
+        if (patient == null || g < 0) {
```

Comment:
Unless we want 0 to have the special meaning of "all", this should be kept as `g <= 0`